docroot/themes/custom/uids_base/templates/layouts/onecol--background/layout--onecol--background.html.twig

```diff
@@ -19,11 +19,10 @@
     'banner--gradient-bottom',
     'banner--gradient-dark',
     'banner--horizontal-left',
-    'banner--large',
+    "banner--#{field_featured_image_size}",
```

Comment:
@GaryRidgway Can you set it to default to `banner--large` and then have it adjust based on the size selection?

Reply:
The select list is defaulted to large on the edit page. would you like it to default here as well or would the default on the edit page be enough for this?

Reply:
Probably something like this instead so existing banners still look good: `field_featured_image_size ? 'banner--' ~ field_featured_image_size : 'banner--large',`

Reply:
This logic is moved into the `uids_base_preprocess_layout` function to keep the logic in the twig template cleaner.